requirements.txt

```
@@ -1,11 +1,81 @@
+#
+# This file is autogenerated by pip-compile with python 3.8
+# To update, run:
+#
+#    pip-compile requirements.in
+#
 aiohttp==3.8.0
+    # via -r requirements.in
+aiosignal==1.2.0
+    # via aiohttp
+async-timeout==4.0.0
+    # via aiohttp
+attrs==21.2.0
+    # via
+    #   aiohttp
+    #   jsonschema
+authlib==0.15.5
+    # via -r requirements.in
+certifi==2021.10.8
+    # via requests
+cffi==1.15.0
+    # via cryptography
+charset-normalizer==2.0.7
+    # via
+    #   aiohttp
+    #   requests
 cryptography==35.0.0
+    # via
+    #   -r requirements.in
+    #   authlib
+elementpath==2.4.0
+    # via xmlschema
+frozenlist==1.2.0
+    # via
+    #   aiohttp
+    #   aiosignal
 gunicorn==20.1.0
+    # via -r requirements.in
+idna==3.3
+    # via
+    #   requests
+    #   yarl
+importlib-resources==5.4.0
+    # via jsonschema
 jsonschema==4.2.1
+    # via -r requirements.in
 motor==2.5.1
+    # via -r requirements.in
+multidict==5.2.0
+    # via
+    #   aiohttp
+    #   yarl
+pycparser==2.21
+    # via cffi
+pymongo==3.12.1
+    # via motor
+pyrsistent==0.18.0
+    # via jsonschema
 python-dateutil==2.8.2
+    # via -r requirements.in
+requests==2.26.0
+    # via -r requirements.in
+six==1.16.0
+    # via python-dateutil
+typing-extensions==3.10.0.2
+    # via async-timeout
+ujson==4.2.0
+    # via -r requirements.in
+urllib3==1.26.7
+    # via requests
 uvloop==0.16.0
+    # via -r requirements.in
 xmlschema==1.8.1
```

```suggestion
xmlschema==1.8.2
```
sorry i updated some dependencies :( that caused conflicts